Top10Words/BookCalculationsLinq.cs

```diff
@@ -1,41 +1,38 @@
-using System.Linq;
-using BenchmarkDotNet.Attributes;
-using BenchmarkDotNet.Engines;
-using BenchmarkDotNet.Jobs;
-
 namespace Top10Words;
 
-public class BookCalculationsLinq
+public class BookCalculationsLinq:IBookCalculations
 {
-    private DataFromBook _dataFromBook;
+    private readonly DataFromBook _dataFromBook;
     private IBookProcessor _bookProcessor;
-    private readonly Consumer _consumer = new Consumer();
     public BookCalculationsLinq()
     {
-        _dataFromBook = new DataFromBook(@"/Users/mariialeos/RiderProjects/Top10Words/Top10Words/Book.txt");
+        var path = "../../../Book.txt";
```

Comment:
I use these constructors in benchmark project.

Reply:
That is a problme of implementation.
Imaging your BookCalculationLinq class is part of your libriary that other developers can use.
That means, I can use your class, use your default constructor and it will fail.
Does it mean, I should ask you to create another constructor with a specific path for me?
No. You should provide an "interface" where I can pass the parameter. And you already have it.

Now, your benchmark test project is "another developer" which is independant from you. He is not able to ask you to create a hardoceded values.

Solution. Just remove your default constructor and pass paratamers from BenchmarkTests project.